fearless/Common/Model/ChainModel.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import RobinHood
+
+struct ChainModel: Codable, Equatable {
+    var chainId: String
+    var name: String
+    var externalApi: ChainExternalApi?
+
+    static func == (lhs: ChainModel, rhs: ChainModel) -> Bool {
+        lhs.chainId == rhs.chainId && lhs.externalApi == rhs.externalApi
```

Comment:
why need to compare external api ? is it really possible to have multiple chain instances with different external apis?